ConcernsCaseWork/Service.TRAMS/Cases/CaseActionEnum.cs

```diff
@@ -2,6 +2,7 @@
 {
 	public enum CaseActionEnum
 	{
+		Decision,
```

Comment:
By inserting the Decision enum here this has changed the value of 0 from meaning DfESupport to meaning Decision.
This makes it a breaking change.
If the enum members are to be defined in alphabetical order, then they should specify a value. Otherwise they can only be appended to the bottom of the member list

Reply:
Not used in Trams, wont be saved.
Will move this file out of the Trams folder